README.md

```diff
@@ -1047,7 +1055,7 @@ Turn off local AEM instance(s).
 
 #### Task `aemRestart`
 
-Turn off then on local AEM instance(s).
+Restarts local AEM instance(s).
```

Comment:
previously it was more precise intentionally (there is restart btn on /system/console/vmstat so that I intentionally made it less confusing)